app/javascript/react/components/FlavorTile.js

```diff
@@ -2,11 +2,9 @@ import React from "react";
 import { Link } from "react-router-dom";
 
 const FlavorTile = (props) => {
-  const { name } = props;
+  const { name, flavorId, regionId } = props;
```

Comment:
see above comments, lets move this dope destructiring into line 4

Reply:
Made recommended change